universe/package.json

```diff
@@ -1,11 +1,11 @@
 {
     "packagingVersion": "3.0",
-    "postInstallNotes": "DC/OS Spark is being installed!\n\n\tDocumentation: https://docs.mesosphere.com/service-docs/spark/\n\tIssues: https://docs.mesosphere.com/support/",
+    "postInstallNotes": "DC/OS Spark is being installed!\n\n\tDocumentation: https://docs.mesosphere.com/current/usage/service-guides/spark/\n\tIssues: https://docs.mesosphere.com/support/",
```

Comment:
I don't think the docs URL needs to be changed here. Same for "postUninstallNotes", "description", and "website" below.